### PR TITLE
gphoto2: 2.5.11 -> 2.5.15

### PR DESCRIPTION
**Pull request by dywedir**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by Mic92** [comment]
@GrahamcOfBorg build gphoto2

File: pkgs/applications/misc/gphoto2/default.nix [hunk 1]
```diff
@@ -3,11 +3,11 @@
 }:
 
 stdenv.mkDerivation rec {
-  name = "gphoto2-2.5.11";
+  name = "gphoto2-2.5.15";
 
   src = fetchurl {
     url = "mirror://sourceforge/gphoto/${name}.tar.bz2";
-    sha256 = "1sgr6rsvzzagcwhc8fxbnvz3k02wr2hab0vrbvcb04k5l3b48a1r";
+    sha256 = "0xsa12k5fz49v8y4h3zahzr427a3ylxaf0k7hybrkp43g4i1lmxf";
```
Comment (vcunat) [inline pkgs/applications/misc/gphoto2/default.nix]: The hash doesn't match, on [Hydra](https://hydra.nixos.org/build/70501543) and locally.  Any idea why it's wrong?

Comment (vcunat) [reply pkgs/applications/misc/gphoto2/default.nix]: E.g., upstream updating it in-place in the meantime?

Comment (dywedir) [reply pkgs/applications/misc/gphoto2/default.nix]: Something wrong with sourceforge

```
curl -LO http://downloads.sourceforge.net/gphoto/gphoto2-2.5.15.tar.bz2
cat gphoto2-2.5.15.tar.bz2:
```
```
<html><head>
<title>SourceForge</title>
<!-- <script src="/js/jquery.com/jquery-1.11.0.min.js"></script> -->
<script src="https://code.jquery.com/jquery-1.11.0.min.js"></script>
<script src="https://sourceforge.net/js/mirrors.js"></script>
<script src="/js/sf.js"></script>
<script>
var DR_loc = DR_parse_hash_url();
if (DR_loc) {
    DR_sf_main(DR_loc);
} else {
    window.location.href = 'https://sourceforge.net/home.html';
}
</script>
</head><body>
<noscript>
We're sorry -- the Sourceforge site is currently in Disaster Recovery mode, and currently requires
the use of javascript to function.  Please check back later.
</noscript>
</body></html>
```

Comment (vcunat) [reply pkgs/applications/misc/gphoto2/default.nix]: Now it seems OK and we got it into our binary cache, too.

**Pull request merged**